featurebyte/api/item_view.py

```diff
@@ -303,8 +303,9 @@ def _are_columns_derived_only_from_event_table(self, column_names: List[str]) ->
         -------
         bool
         """
+        # FIXME: This method is may not work correctly when the column is from SCD table joined with EventTable
```

Comment:
Indeed this is an existing issue that should be fixed. Created ticket to track: https://featurebyte.atlassian.net/browse/DEV-1945